tfs/handler.py

```diff
@@ -203,7 +205,7 @@ def write_tfs(
 
     LOGGER.debug(f"Attempting to write file: {tfs_file_path.name} in {tfs_file_path.parent}")
     with tfs_file_path.open("w") as tfs_data:
-        tfs_data.write("\n".join((headers_str, colnames_str, coltypes_str, data_str)))
+        tfs_data.write("".join((headers_str, colnames_str, coltypes_str, data_str)))
```

Comment:
You could have saved some work (and I think it's still more elegant) with :

```
tfs_data.write("\n".join((str_ for str_ in (headers_str, colnames_str, coltypes_str, data_str) if str_)))
```

Reply:
(or in a second line)

Reply:
Done in e3f5be9